packages/daemon/src/mail.js

```diff
@@ -486,20 +467,18 @@ export const makeMailboxMaker = ({
         throw new Error(`Unknown pet name for party: ${recipientName}`);
       }
       const recipientController =
-        /** @type {import('./types.js').Controller<>} */ (
-          await provideControllerForFormulaIdentifier(
-            recipientFormulaIdentifier,
-          )
+        await provideControllerForFormulaIdentifierAndResolveHandle(
+          recipientFormulaIdentifier,
         );
-
       const recipientInternal = await recipientController.internal;
-      if (recipientInternal === undefined) {
+      if (recipientInternal === undefined || recipientInternal === null) {
```

Comment:
Why can this be `null` and not just `undefined`?

Reply:
just appeasing type checker